main.js

```diff
@@ -13,6 +13,275 @@ toggle.addEventListener('click', () => {
 
   navlink.classList.toggle('active');
 });
+const cardsData = [
+  {
+    class: 'ttl',
+    class2: 'cards',
+    heading: 'My Recent Works',
+  },
+  {
+    class: ['img-hdr', 'langs'],
+    divCls: [
+      'card cd1',
+      'card cd2',
+      'card cd3',
+      'card cd4',
+      'card cd5',
+      'card cd6',
+    ],
+    listcls: 'cd-links',
+    heading: ['Multi-Post Stories', 'Gain+Glory'],
+    lists: ['Ruby on Rails', 'css', 'Javascript', 'html'],
+    btn: 'See Project',
+    btnCls: 'cardBtn',
+  },
+];
+const popupData = [
+  {
+    id: 'popup',
+    divCls: ['popimg', 'poptext', 'popHead', 'topBtns', 'btns'],
+    classes: [
+      'fa-solid fa-x close',
+      'imgMob',
+      'imgDesk',
+      'popTtl',
+      'btn seeLive',
+      'btn seeLink',
+      'popLinks',
+      'link',
+      'popPara',
+    ],
+    btns: ['See Live', 'See source'],
+    images: ['./assets/mobile.png', './assets/popImg.png'],
+    h2: 'Keeping track of hundreds of components',
+    btnLinks: [
+      'https://prg-04.github.io/Portfolio_',
+      'https://github.com/prg-04/Portfolio_',
+    ],
+    lists: [
+      'Codekit',
+      'Github',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry, Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release",
+  },
+];
```

Comment:
- [ ] As per the project requirement, you are expected to use an array to store a javascript object where each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie:

```javascript
const projects = [{
    id: 1
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 2
    name: "Helicopter game",
    description: "A Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 3
    name: "Cars racing game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]

// Then you will map through these projects array and display each project. 
```

See requirement 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/218054980-f296b6ff-6b63-4aee-be33-efd34c075424.png)